test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```

Comment:
This file is not necessary, you can delete it

Reply:
i could not find this file in the sublime